src/py/flwr/server/serverapp/app.py

```diff
@@ -108,6 +106,7 @@ def flwr_serverapp() -> None:
 
     certificates = _try_obtain_certificates(args)
 
+    log(INFO, "Starting Flower ServerApp")
```

Comment:
@chongshenng should we do this before loading certificates? imo, it's good to show the log early to show users that the system is alive

Reply:
Yes, good point!